src/Spryker/Zed/AppKernel/Communication/Plugin/AppKernel/TriggerAppConfigUpdatePostInstallTaskPlugin.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\AppKernel\Communication\Plugin\AppKernel;
+
+use Generated\Shared\Transfer\AppConfigCriteriaTransfer;
+use Spryker\Zed\AppKernelExtension\Dependency\Plugin\PostInstallTaskPluginInterface;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \Spryker\Zed\AppKernel\Business\AppKernelFacadeInterface getFacade()
+ * @method \Spryker\Zed\AppKernel\Persistence\AppKernelRepositoryInterface getRepository()
+ * @method \Spryker\Zed\AppKernel\AppKernelConfig getConfig()
+ * @method \Spryker\Zed\AppKernel\Communication\AppKernelCommunicationFactory getFactory()
+ */
+class TriggerAppConfigUpdatePostInstallTaskPlugin extends AbstractPlugin implements PostInstallTaskPluginInterface
```

Comment:
What is this used for?

Reply:
To ensure that any configuration changes made at any stage of development, such as adding channels, are taken into account during deployment.
With this command, all tenants will automatically receive updates about any channel added/removed in any application.